index.html

```diff
@@ -50,7 +47,276 @@ <h6>LET'S CONNECT</h6>
+                    See Project
+                </button>
+        </section>
+ 
+     </section>
+
+     <section class="works-three">
+       
+        <section class="image">
+            <img src="imagethree.png" alt="imageone">
+        </section>
+        
+        <section class="project-title">
+         <h2>Tonic</h2>
+         <section class="small-title">
+             <p>CANOPY</p>
+             <section class="list">
+                 <ul>
+                 <li>Back End Dev</li>
+                 <li>2015</li>
+                 </ul>
+             </section>
+                 
+        </section>
+ 
+        <section class="describe-project"> 
+           <p>
+             A daily selection of privately personalized reads; no accounts or sign-ups required.
+           </p>
+        </section>
+ 
+        <section class="small-buttons">
+            <button>  
+                html
+             </button>
+             <button>
+                 css
+             </button>
+             <button>
+             javascript
+             </button>
+        </section>
+ 
+        <section class="big-button">
+                <button>
+                    See Project
+                </button>
+        </section>
  
+     </section>
+
+     <section class="works-four">
+       
+        <section class="image">
+            <img src="imagefour.png" alt="imageone">
+        </section>
+        
+        <section class="project-title">
+         <h2>Multi-Post <br> Stories</h2>
+         <section class="small-title">
+             <p>CANOPY</p>
+             <section class="list">
+                 <ul>
+                 <li>Back End Dev</li>
+                 <li>2015</li>
+                 </ul>
+             </section>
+       </section>
+
+ 
+        <section class="describe-project"> 
+           <p>
+             A daily selection of privately personalized reads; no accounts or sign-ups required.
+           </p>
+        </section>
+ 
+        <section class="small-buttons">
+                    <button>  
+                        html
+                     </button>
+                     <button>
+                         css
+                     </button>
+                     <button>
+                     javascript
+                     </button>
+        </section>
+ 
+        <section class="big-button">
+                <button>
+                    See Project
+                </button>
+        </section>
+ 
+     </section>
+
+  <section class="aboutme">
+ 
+    <section class="mytitle">
+        <p>About <br> Myself</p>
+    </section>
+
+    <section class="describe-me">
+      <p>
+        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+    </section>
+
+    <section class="connect">
+          <p>LET'S CONNECT</p>
+          <ul>
```

Comment:
- [ ] Please use a header tag here instead of a paragraph. You could use a `<h3>` or `<h4>`